python/doc/examples/meta_modeling/kriging_metamodel/plot_kriging_branin_function.py

```diff
@@ -62,10 +63,10 @@
 
 
 # %%
-# Definition of the Kriging metamodel
-# -----------------------------------
+# Definition of the Gaussian Process fitter (GP) metamodel
+# --------------------------------------------------------
```

Comment:
Optimization of the GP hyperparameters ?

Reply:
In this part, the GP hyperparameters are optimized and the GPR is conditioned to the data. I suggest : 

"Optimization of GP hyperparameters and conditioning to data"